src/Core/Commander/Providers/PanoramicProvider.js

```diff
@@ -72,24 +70,18 @@
      * @param {type} distance
      * @returns {Promise}
      */
-    PanoramicProvider.prototype.getMetaDataFromPos = function(longitude, latitude, distance){
-
-        var that = this;
-        if(!_panoramicsMetaData){
+    PanoramicProvider.prototype.getMetaDataFromPos = function(longitude, latitude/*, distance*/){
```

Comment:
OK, removed. (and directly amended the commit, given the nature and scope of the change)